GDM.HW4.OOP.Classes.Task1/Program.cs

```diff
@@ -0,0 +1,143 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GDM.HW4.OOP.Classes.Task1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            const string laptop = "Laptop";
+            const string server = "Server";
+            const string plazmTV = "PlasmTV";
+            const string ledTV = "LED TV";
+            const string pleyer = "Pleyer Device";
+            
+            LapTop Acer = new LapTop("Acer", 65, 8, 2.1);
+            LapTop MSI = new LapTop("MSI", 85, 16, 2.9);
+            Console.WriteLine(laptop + " 1:\n" + Acer.GetDescription());
+            Console.WriteLine(laptop + " 2:\n" + MSI.GetDescription());
+
+            Server Intel = new Server("Intel", 200, 120, 8);
+            Server AMD = new Server("Intel", 300, 140, 12);
+            Console.WriteLine(server + " 1:\n" + Intel.GetDescription());
+            Console.WriteLine(server + " 2:\n" + AMD.GetDescription());
+
+            PlasmTV Samsung = new PlasmTV("Samsung", 300, 32, "1920x1080");
+            PlasmTV LG = new PlasmTV("LG", 500, 50, "3840x2160");
+            Console.WriteLine(plazmTV + " 1:\n" + Samsung.GetDescription());
+            Console.WriteLine(plazmTV + " 2:\n" + LG.GetDescription());
+
+            LEDTV Sharp = new LEDTV("Sharp", 350, 40, 60);
+            LEDTV Xiaomi = new LEDTV("Xiaomi", 700, 65, 75);
+            Console.WriteLine(ledTV + " 1:\n" + Sharp.GetDescription());
+            Console.WriteLine(ledTV + " 2:\n" + Xiaomi.GetDescription());
+
+            PleyerDevice Sony = new PleyerDevice("Sharp", 350, true, true);
+            PleyerDevice Panasonic = new PleyerDevice("Xiaomi", 700, true, false);
+            Console.WriteLine(pleyer + " 1:\n" + Sony.GetDescription());
+            Console.WriteLine(pleyer + " 2:\n" + Panasonic.GetDescription());
+
+
+            Console.ReadLine();
+        }
+    }
+    class LapTop
+    {
+        public string Name;
```

Comment:
Actually you need to protect these fields from rewriting from outside. Anyway you pass info to constructor.